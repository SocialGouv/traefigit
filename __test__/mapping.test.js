const convertMapping = require("../src/convertMapping");

const sampleMapping = require("../mapping.sample.json");

test("should work", () => {
  expect(convertMapping(sampleMapping)).toMatchSnapshot();
});
