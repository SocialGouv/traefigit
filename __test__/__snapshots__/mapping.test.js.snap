// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should work 1`] = `
Object {
  "backends": Object {
    "website-doc": Object {
      "servers": Object {
        "server0": Object {
          "url": "https://hello.github.io",
        },
      },
    },
    "website-metabase": Object {
      "servers": Object {
        "server0": Object {
          "url": "http://1.2.3.4:8080",
        },
      },
    },
  },
  "frontends": Object {
    "website-doc": Object {
      "backend": "website-doc",
      "routes": Object {
        "route0": Object {
          "rule": "Host:website-doc.domain.fr; AddPrefix: /website-doc",
        },
      },
    },
    "website-metabase": Object {
      "backend": "website-metabase",
      "basicAuth": Array [
        "code:$apr1$gSzefXeHY$YhplnNO25blGcAx/vb0",
      ],
      "passHostHeader": true,
      "routes": Object {
        "route0": Object {
          "rule": "Host:website-metabase.domain.fr",
        },
      },
    },
  },
}
`;
